docs/testplan/BGP-Speaker-2byte-to-4byte-ASN.md

```diff
@@ -0,0 +1,46 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+## Overview
+The purpose of this test is to validate communication between a 2 byte ASN device and a 4 byte ASN device.
+### Scope
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is not to test specific API, but functional testing of BGP configuration on SONIC system.
+
+### Testbed
+The test is developed and tested with Sonic t0 testbed setup and CEOS neighbors.
+
+## Setup configuration
+This test doesn't required any configurations from outside. But intenally it will modify the ASN of BGPVac and BGPSLBPassive groups in which the new neighboars are being created dynamically. Ensure the "bgp_speaker_route_4bASN.j2" file is available under sonic-mgmt/tests/bgp/templates.
+
+## Test
+Configure testbed to emulate production.
+Configure IXIA/Spirent with a 4-byte ASN and peer with DUT.
```

Comment:
why ixia here? are we using ixia here? if not, please remove from the description.